src/bp/admin/ui/src/app/index.tsx

```diff
@@ -1,19 +1,25 @@
-import { Alignment, Icon, Navbar } from '@blueprintjs/core'
+import { Icon } from '@blueprintjs/core'
 import { lang, TokenRefresher } from 'botpress/shared'
 import cx from 'classnames'
 import React, { FC, Fragment, useEffect } from 'react'
 import { connect, ConnectedProps } from 'react-redux'
 import { Link } from 'react-router-dom'
+import SplitPane from 'react-split-pane'
 
 import api from '~/app/api'
 import { fetchLicensing } from '~/management/licensing/reducer'
 import { fetchProfile } from '~/user/reducer'
+import BottomPanel from './BottomPanel'
 
 import CommandPalette from './CommandPalette'
 import EventBus from './EventBus'
+import Header from './Header'
 import Menu from './Menu'
 import { AppState } from './rootReducer'
 import style from './style.scss'
+import { toggleBottomPanel } from './uiReducer'
+
+const EXPANDED_PANEL_HEIGHT = 200
```

Comment:
200 refers to the size of the top panel (not the bottom one). I kept the same values as the studio since we are used to it, most of the time someone will change the height and it will be lower than 175